src/command_modules/race_control.js

```diff
@@ -375,7 +375,7 @@ export const commands = {
 		},
 	},
 	raceCategory: {
-		names: [ "category" ],
+		names: [ "category", "c" ],
```

Comment:
I'm not so sure about this change.
`c` and `l` were removed because it lead to a very obfuscated "UI". What I mean is that it might be too much for new users to learn all these abbreviations and they might not even understand other people's interactions with the bot.

Only things like `r` or `d` (which are time sensitive or need to be typed very often) were kept for that reason.

Reply:
Yeah I agree with this